styles.css

```diff
@@ -333,21 +475,36 @@ h2 {
 #about-me {
   /* display: none; */
```

Comment:
[OPTIONAL] To keep your codebase clean and more proffessional whiles reducing size, kindly remove all unused code from your codebase :wink: 
You can find other instances of this on lines 112, 208, 295, 476, 579, 676, etc

Reply:
`display: none` I use with every section which I'm not working on while I build a page. 
When you make any changes, the page reloads, and if you are not on the top of a page, you need to scroll down to see those changes. 
That's where hiding sections helps to save time

Reply:
The **general rule** is that you should remove code that you don't need any more, to _keep the codebase as clean as possible_. So, if you don't expect your code to be re-used in the 'near future', it is best practice to remove them.
However, in this case, there is good reason to expect your code to be used in the next milestone as you just explained :+1: . So, commenting it out is probably your best bet. After the final milestone of the project and you don't use it afterall, it will be better to remove them :wink:

Reply:
> `display: none` I use with every section which I'm not working on while I build a page. When you make any changes, the page reloads, and if you are not on the top of a page, you need to scroll down to see those changes. That's where hiding sections helps to save time

I like the concept. Maybe I will also start using it to keep me from always scrolling up and down. Just need to remove them, though, after the project, for sake of best practices. :1st_place_medal:

Reply:
Love that it's already your third comment on it : )

To be efficient is to save your time everywhere you can.
So, I'm happy to share something with you from the first project in my life.